test/test_cuda.py

```diff
@@ -1088,6 +1088,40 @@ def test_type_conversions_same_gpu(self):
     def test_neg(self):
         _TestTorchMixin._test_neg(self, lambda t: t.cuda())
 
+    def test_bitwise_not(self):
```

Comment:
I think the CUDA tests should basically be the same as the CPU test. Instead of writing them twice (once in test_torch.py and once in test_cuda.py) you can either:

- follow the pattern of `test_neg` above
- Test on "torch.testing.get_all_device_types()" in test_torch.py like here:
https://github.com/pytorch/pytorch/blob/b76877728a1f06c936a63e7028651561ca3248fc/test/test_torch.py#L3334-L3335

Reply:
I adopted the test_neg way